ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TipsLayout.kt

```diff
@@ -0,0 +1,96 @@
+package org.cru.godtools.tutorial
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
```

Comment:
Android Studio by default uses wildcard imports even though it's considered a best practice to not use them. To avoid Android Studio constantly switching your imports to them you can modify the code style settings in preferences.

In the Android Studio preferences you can go here to change the setting to "Use single name import"

![Screen Shot 2022-06-10 at 5 35 03 PM](https://user-images.githubusercontent.com/47818/173154342-4bfcc8ed-9bbb-44b7-a96f-5288fc3ee3ac.png)